docs/docs/user-guide/appendix/FAQ.md

```diff
@@ -0,0 +1,51 @@
+# Frequently Asked Questions
+
+### Is BioNeMo free to use?
+
+Yes, BioNeMo is free to use. BioNeMo code is licensed under the Apache 2.0 License. The Apache 2.0 License is a
```

Comment:
let's say bionemo framework explicitly everywhere instead of bionemo. I think bionemo includes framework, NIMs etc.

Reply:
Done